sunpy/map/mapbase.py

```diff
@@ -195,21 +196,10 @@ def __init__(self, data, header, plot_settings=None, **kwargs):
         self._validate_meta()
         self._shift = SpatialPair(0 * u.arcsec, 0 * u.arcsec)
 
-        if self.dtype == np.uint8:
-            norm = None
-        else:
-            # Put import here to reduce sunpy.map import time
-            from matplotlib import colors
-            norm = colors.Normalize()
-
         # Visualization attributes
-        self.plot_settings = {'cmap': 'gray',
-                              'norm': norm,
-                              'interpolation': 'nearest',
-                              'origin': 'lower'
-                              }
-        if plot_settings:
-            self.plot_settings.update(plot_settings)
+        self.plot_settings = plot_settings
+        self.cmap = cmap
```

Comment:
Is the original way of providing cmap and norm using `plot_settings` still supported?

Reply:
@vn-ki If `plot_settings` is passed in `__init__`, then the setter updated the `cmap` and `norm`, if these are passed in `plot_settings`